android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -274,7 +274,7 @@ class RNIapModule(
             promise
         ) {
             val items = WritableNativeArray()
-            billingClient.queryPurchasesAsync(
+            billingClient?.queryPurchasesAsync(
```

Comment:
Same question here. Shouldn't we reject a case where `billingClient` is null?

Reply:
same, now it's never null